pkg/leap_mx

```diff
@@ -21,7 +21,8 @@ case "$1" in
                           --pidfile=$pidfile \
                           --rundir=$rundir \
                           --python=$file \
-                          --logfile=$logfile
+                          --logfile=$logfile \
+			  --syslog
```

Comment:
The ultimate nitpick comment: shouldn't that be one space more to the right so it lines up with the stack above?